### ISSUE TITLE
Disseminated candidiasis

### ISSUE THREAD

**Issue by Marwaabdelhakim** [obophenotype/human-phenotype-ontology#11132]
**Preferred term label:**
Disseminated candidiasis

**Synonyms**
hepatosplenic candidiasis

**Definition (free text, please give PubMed ID)**
 severe form of disseminated fungal infection that commonly affects the liver, spleen, and kidneys

**Parent term (use hpo.jax.org/app)**

Invasive fungal infection |disseminated fungal infection 
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
[0000-0001-6816-2119](https://orcid.org/0000-0001-6816-2119)

**Comment by pnrobinson** [comment]
@Marwaabdelhakim 
Adding new term: 
- ID: HP:6001283
- label: Disseminated candidiasis
- parents: [Invasive fungal infection (HP:0020101)](https://hpo.jax.org/app/browse/term/HP:0020101)
- pmid: PMID:38448809
- def: A systemic fungal infection by Candida species (most commonly, C. albicans).
- comment: Candida species are colonizers of the human skin and gastrointestinal (GI) tract, existing as part of the microbiome in many healthy individuals. The pathophysiology of disseminated candidiasis in immunocompromised patients is poorly understood but thought to be related to chemotherapy-induced mucositis and profound neutropenia. This allows for Candida species to enter the bloodstream where the organisms can translocate into the portal venous system and seed to the liver and spleen, which explains why the lesions are primarily seen in these two organs. Renal, pulmonary, and other organ involvement have also been documented but not as common.
- orcid: orcid:0000-0001-6816-2119
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11132
- synonyms: Hepatosplenic candidiasis(NONE)
